src/plugins/Mentions/MentionEditor.jsx

```diff
@@ -10,6 +10,7 @@ import { setPluginOptions } from '@plone/volto-slate/actions';
 import { MentionSchema } from './schema';
 import { getMentionWidget } from './utils';
 
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
@ana-oprea We should avoid as much as possible `eslint-disable` comments to fix eslint issues

Reply:
@avoinea I commited the fix